packages/wc-compass/src/components/rich-text-editor/rte-toolbar.js

```diff
@@ -0,0 +1,163 @@
+export class CdgRichTextEditorToolbar extends HTMLElement {
+  constructor() {
+    super()
+    this.viewMoreButton = document.createElement('button')
+    this.viewMoreButton.classList.add('more-button')
+    this.viewMoreButton.innerHTML =
+      '<cdg-icon name="activity" source="host"></cdg-icon>'
+    this.appendChild(this.viewMoreButton)
+  }
+
+  set editor(value) {
+    this._editor = value
+  }
+  get editor() {
+    return this._editor
+  }
+  buttonList
+  viewMoreButton
+  headingSelector
+  colorSelector
+  connectedCallback() {
+    this.buttonList = document.querySelectorAll('button.cdg-rte-buttons')

```

Comment:
Use this.querySelectorAll instead or passing the anchor from the outside. Imagine that if we have multiple rich text editors on the screen, this toolbar can handle other toolbar.